modules/module/server-ts/ServerModule.ts

```diff
@@ -20,8 +20,10 @@ export interface ServerModuleShape extends CommonModuleShape {
   createResolversFunc?: Array<(pubsub: PubSub) => IResolvers>;
   createContextFunc?: Array<(props: CreateContextFuncProps) => { [key: string]: any }>;
   // Middleware
-  beforeware?: Array<(app: Express) => void>;
-  middleware?: Array<(app: Express) => void>;
+  beforeware?: Array<(app: Express, module: object) => void>;
```

Comment:
`module: object`,
I think it should be `modules`
and it should be `modules: ServerModuleShape`, or `modules: ServerModule`

Reply:
Done